README.md

```diff
@@ -113,14 +113,14 @@ bundler.generateManifest([target]).then((manifest) => {
   bundler.bundle(manifest).then((bundles) => {
     /**
       * do stuff here
-      */      
+      */
   });
 });
 ```
 
 ## Caveats
 
-Because HTML Imports changes the order of execution scripts can have, polymer-bundler has to make a few compromises to achieve that same script 
+Because HTML Imports changes the order of execution scripts can have, polymer-bundler has to make a few compromises to achieve that same script
```

Comment:
Fair point.  This was worded weirdly and wrongly-- This was meant to describe the problem of an html import forcing us to move imperatives like scripts and styles out of the head down into a hidden div after the inlined import. Rewritten.